packages/angular/migrations.json

```diff
@@ -1259,6 +1259,99 @@
           "alwaysAddToPackageJson": false
         }
       }
+    },
+    "14.2.0-beta.4": {
+      "version": "14.2.0-beta.4",
```

Comment:
We can update the version of the previous migration, we don't need a separate block for every `rc`.